server/author/tests/test_create_author_endpoint.py

```diff
@@ -62,6 +70,85 @@ def test_create_user_with_password_too_short(self):
             'password':'abc',
         }
 
-        res = self.client.post(SIGNUP_USER_URL, payload)
+        res = self.client.post(CREATE_USER_URL, payload)
+
+        self.assertEqual(res.status_code, status.HTTP_400_BAD_REQUEST)
+
+class TestAuthAuthorEndpoint(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_auth_author_endpoint_to_return_token(self):
```

Comment:
could we also add docstrings for each of these unit tests? It makes it easier to debug in the console later
Something like

def test_auth_author_endpoint_to_return_token(self):
   """Test if author endpoint returns token properly"""

Reply:
addressed but GitHub did not pick it up because it was a new line 😭.